src/main/webapp/WEB-INF/web.xml

```diff
@@ -17,6 +11,7 @@
 	<servlet-mapping>
 		<servlet-name>synapseAwsConsoleLogin</servlet-name>
 		<url-pattern>/synapse</url-pattern>
+    	<url-pattern>/</url-pattern>
```

Comment:
should this be indented?  and also i distinctly remember that tomcat doesn't allow multiple <url-pattern> inside of a servlet-mapping?  https://github.com/Sage-Bionetworks/synapse-login-scipoolprod/pull/6

Reply:
> should this be indented?

I updated the whitespace.

> tomcat doesn't allow multiple <url-pattern> inside of a servlet-mapping? 

I think you misunderstood the error message.  It doesn't say that one mapping can't refer to two servlet patterns, but rather than two different mappings can't refer to the *same* servlet pattern. That's why I removed the mapping from `default` to `/` in this PR.